src/blurredLocation.js

```diff
@@ -61,8 +67,7 @@ BlurredLocation = function BlurredLocation(options) {
     });
     var geometry = Blurred.responseJSON.results[0].geometry.location;
     options.map.setView([geometry.lat, geometry.lng],options.zoom);
```

Comment:
Best move this into the onComplete method and name it onGeocodeString, maybe? But can we make it the default of the function and not the whole class? Like, `geocodeStringAndPan (string, onComplete)`, then inside the function we do `onComplete = onComplete || function onComplete (geometry) { ... }`